cli/setup_cmd.go

```diff
@@ -172,7 +177,9 @@ func setupRun(cmd *cobra.Command, args []string) {
 
 		// !useIPTables is used, because when the iptables step is enabled, we want
 		// --iptables=false for the docker daemon.
-		if err := docker.Setup(fsClient, systemdClient, fetchClient, overlayMountPoint, dockerVersion, privateRegistry, !useIPTables, startDaemons, useOverlay); err != nil {
+		enableDockerIptables := !useDockerIptableRules
```

Comment:
This is confusing. I think you don't need to map `useDockerIptableRules` to `enableDockerIptables`. And the naming is almost identically. 

 * `useDockerIptableRules=true` means that not docker should control iptables but yochu sets some rules for docker
 * `enableDockerIptables=true` means that docker should control iptables